src/System.IO.FileSystem.Watcher/src/win/project.json

```diff
@@ -1,25 +1,26 @@
 {
```

Comment:
I'm not sure I understand why there is a separate project.json for netstandard1.7 if this is also includded on the one under src/ . Is there a reason why we can't clean this up?